pom.xml

```diff
@@ -165,7 +166,7 @@
                             <goal>validate</goal>
                         </goals>
                         <configuration>
-                            <skipFormatting>false</skipFormatting>
+                            <skipFormatting>${formatter.skip}</skipFormatting>
```

Comment:
You can directly remove the config section because the formatter.skip is the default property name used by the plugin and we are already passing it as a default property to the test stage https://github.com/mulesoft/mule-runtime-pipeline/blob/master/resources/config/default.yaml#L56

Reply:
great, will do!